core/store/models/job_spec.go

```diff
@@ -234,6 +253,38 @@ func (t Topics) Value() (driver.Value, error) {
 	return string(j), nil
 }
 
+// Feeds holds all flux monitor feed URLs, serializing into the db
+// with ; delimited strings.
+type Feeds []string
+
+// Scan hydrates the current Feeds value with the passed in value, usually a
+// string from an underlying database.
+func (f *Feeds) Scan(value interface{}) error {
+	str, ok := value.(string)
+	if !ok {
+		return fmt.Errorf("Unable to convert %v of %T to Feeds", value, value)
+	}
+
+	arr := strings.Split(str, ";")
+	collection := []string{}
+	for _, entry := range arr {
+		if entry != "" {
+			collection = append(collection, entry)
+		}
+	}
+	*f = collection
+	return nil
+}
+
+// Value returns this instance serialized for database storage.
+func (f Feeds) Value() (driver.Value, error) {
+	if len(f) == 0 {
+		return nil, nil
+	}
+	str := strings.Join(f, ";")
```

Comment:
This could be done with csv pkg setting the 'Comma' field to ';' to avoid issues related to escaping characters.  I'm a little concerned about funny values making it in and causing problems.

Reply:
What do you think?

Reply:
Yeah, it's a good idea. It does imply support for things like integers,floats,multiline strings, etc, during deserialization, which we don't want in this instance. But some of these things should be caught during validation, before persisting to sql. Our validation for Feeds is a little light, I'll add to it.

Reply:
Did URL validation on the contents, so it'll always be a valid URL, which limits our new to escape characters. I think we're fine with strings.Join for now and don't need the sophistication of `csv` delimiting with all their various types.

Reply:
Valid URLs can contain commas and semicolons.

Reply:
We now serialize using JSON encoding. We opted for JSON over CSV @felder-cl because code was already written for it to handle deserialization from a job spec.

Reply:
@j16r the standard library would have dealt with these cases sanely